README.md

```diff
@@ -78,7 +78,7 @@ Having trouble with codecentric's Spring Boot Admin? We’d like to help!
 ## Trademarks and licenses
 The source code of codecentric's Spring Boot Admin is licensed under [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 
-Spring, Spring Boot and Spring Cloud are trademarks of [Pivotal Software, Inc.](https://pivotal.io/) in the U.S. and other countries.
+Spring, Spring Boot and Spring Cloud are trademarks of [VMware, Inc.](https://tanzu.vmware.com/) in the U.S. and other countries.
```

Comment:
Hi @torrespro and thanks for your proposal. However, we are not sure if this change would be correct. The trademark guidelines are still pointing to pivotal: https://spring.io/trademarks

Besides that, the link at the bottom of the spring.io site points to `www.vmware.com`, not `tanzu.vmware.com`

Can you provide any reference/source saying that we should reference tanzu/vmware?